### DISCUSSION TITLE
Installer wont run

### DISCUSSION THREAD

**Discussion by Sinedrone-Sentinel** [FreeCAD/FreeCAD#12047] [Discussions that will turn in to issues but need refining]
Like the title says. I download the windows installer, double click and nothing.

**Comment by chennes** [comment]
For 0.21.2?

**Reply by Sinedrone-Sentinel** [reply]
I guess.....whatever version is available for download on the website.

**Reply by chennes** [reply]
What version of Windows are you on?

**Reply by Sinedrone-Sentinel** [reply]
10....there is something wrong with the download available for windows on the website.  I just found the same named one here

https://github.com/FreeCAD/FreeCAD/releases/tag/0.21.2

and downloaded it and this one works

**Reply by chennes** [reply]
It should be the same file -- can you check to see if the filenames match? It's possible I have a typo on the download page or something like that.

**Reply by Sinedrone-Sentinel** [reply]
I will check...it downloads though. so not sure its a "naming" issue

**Reply by Sinedrone-Sentinel** [reply]
Apparently, the one on your website is pointing to 21.1???

**Reply by chennes** [reply]
This is the link I am seeing on the website:
https://github.com/FreeCAD/FreeCAD/releases/download/0.21.2/FreeCAD-0.21.2-WIN-x64-installer-1.exe

**Reply by Sinedrone-Sentinel** [reply]
oh wtf!!

**Reply by Sinedrone-Sentinel** [reply]
I been HAD!

**Reply by chennes** [reply]
Yeah, better check your antivirus, that download could have been anything.

**Reply by Sinedrone-Sentinel** [reply]
I have active protection....interesting nothing popped up though.  did another scan just to be safe and nothing....if I ran that "Fake" one enough times it would show an error message ... dunno

**Reply by Sinedrone-Sentinel** [reply]
jesus....First time since the late 90's that I have been fooled...I'm usually so careful with sites.  But this one, someone spent some time on it.

**Reply by chennes** [reply]
Hopefully that means you are in the clear. At any rate, our real site is at https://freecad.org (for future reference).

**Reply by Sinedrone-Sentinel** [reply]
ORG...got it. Thanks and I'm really glad your links aren't broken LOL

**Reply by chennes** [reply]
Me too! Have a good one.

**Comment by Sinedrone-Sentinel** [comment]
Check out the link handler in bottom left corner


![image](https://github.com/FreeCAD/FreeCAD/assets/52118602/0afc46cb-8799-45ef-acaa-a32be74c6ef6)

**Reply by chennes** [reply]
Oh, you are on a scam website: we don't own freecadus.net -- I'll get that reported.

**Comment by Sinedrone-Sentinel** [comment]
feel free to close this or...put it in the Hall of Shame...or what ever.  Thank you so much for checking in on this!
